src/htcp.cc

```diff
@@ -456,13 +456,19 @@ static ssize_t
 htcpBuildClrOpData(char *buf, size_t buflen, htcpStuff * stuff)
 {
     unsigned short reason;
+    ssize_t s;
```

Comment:
Please declare as late as possible and use AAA/const. Ignore the fact that old Squid code does not.